packages/fxa-settings/src/pages/ResetPasswordConfirmed/index.tsx

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Localized } from '@fluent/react';
+import { usePageViewEvent } from '../../lib/metrics';
+
+type ResetPasswordConfirmedProps = {
+  serviceName: String;
+};
+
+// eslint-disable-next-line no-empty-pattern
```

Comment:
I don't think this is needed. There is an assignment below.

Reply:
Thanks for catching that Dan!